kafka-rest/src/main/java/io/confluent/kafkarest/v2/KafkaConsumerReadTask.java

```diff
@@ -125,27 +124,27 @@ public boolean doPartialRead() {
         );
         finish();
       }
-
-      return true;
     } catch (Exception e) {
       finish(e);
       log.error("Unexpected exception in consumer read task id={} ", this, e);
-      return false;
     }
   }
 
   public boolean isDone() {
     return finished;
   }
 
-  private void finish() {
+  void finish() {
     finish(null);
   }
 
   private void finish(Exception e) {
-    log.trace("Finishing KafkaConsumerReadTask id={} exception={}", this, e);
+    log.trace("Finishing KafkaConsumerReadTask id={}", this);
```

Comment:
We should still include the exception, we just don't need a substitution for it.
```java
log.trace("Finishing KafkaConsumerReadTask id={}", this, e);
```

Reply:
ah, gotcha